draft-ietf-quic-qpack.md

```diff
@@ -715,6 +714,132 @@ None.
+| 41    | content-type                     | text/css                                                                                               |
+| 42    | content-type                     | text/javascript                                                                                        |
+| 43    | content-type                     | application/json; charset=utf-8                                                                        |
+| 44    | content-type                     | text/javascript; charset=utf-8                                                                         |
+| 45    | content-type                     | application/x-javascript                                                                               |
+| 46    | server                           |                                                                                                        |
+| 47    | user-agent                       |                                                                                                        |
+| 48    | accept                           | \*/\*                                                                                                  |
+| 49    | cache-control                    | max-age=0                                                                                              |
+| 50    | cache-control                    | no-cache                                                                                               |
+| 51    | cache-control                    | max-age=2592000                                                                                        |
+| 52    | cache-control                    | public, max-age=31536000                                                                               |
+| 53    | cache-control                    | no-cache, no-store, must-revalidate                                                                    |
+| 54    | cache-control                    | max-age=604800                                                                                         |
+| 55    | accept-encoding                  | gzip, deflate, br                                                                                      |
+| 56    | accept-encoding                  | gzip, deflate                                                                                          |
+| 57    | vary                             | accept-encoding                                                                                        |
+| 58    | vary                             | origin                                                                                                 |
+| 59    | vary                             | accept-encoding,user-agent                                                                             |
```

Comment:
There's no space after the comma. I know that we're basing this on traffic currently seen, but once this is the static table, implementations are going to adjust what they emit to match, making this a pseudo-standard. So I'd suggest either adding a space or dropping this one.